media/server/gstplayer/source/tasks/generic/AttachSource.cpp

```diff
@@ -340,10 +339,11 @@ void AttachSource::reattachAudioSource(GstCaps *caps, const std::string &strCaps
         bool audioAac{oldCapsStr.find("audio/mpeg") != std::string::npos};
         bool svpEnabled{true}; // assume always true
         bool retVal{false};    // Output param. Set to TRUE in rdk_gstreamer_utils function stub
-
+        std::shared_ptr<firebolt::rialto::wrappers::AudioAttributesPrivate> audioAttributes = createAudioAttributes();
```

Comment:
I think we should fail if the return is nullptr, performAudioTrackCodecChannelSwitch may not expect a nullptr.